jupyter-lib/lib/src/main/kotlin/jupyter/kotlin/ScriptTemplateWithDisplayHelpers.kt

```diff
@@ -20,7 +20,7 @@ abstract class ScriptTemplateWithDisplayHelpers(
 
     fun EXECUTE(code: String) = host.scheduleExecution(CodeExecution(code).toExecutionCallback())
 
-    fun USE(library: LibraryDefinition) = host.addLibrary(library)
+    fun USE(library: LibraryDefinition) = host.scheduleExecution { addLibrary(library) }
```

Comment:
If we try to add library immediately, recursive execution happens - execution DURING another execution. We don't know how to handle it, because we use previous script instances for evaluation, and it's not clear what to do without them. So we just schedule the execution to the end of the current execution.